src/components/KTableData/KTableData.vue

```diff
@@ -258,11 +258,11 @@ const hasInitialized = ref<boolean>(false)
 
 const defaultFetcherProps = {
   pageSize: pageSize.value,
-  page: 1,
-  query: '',
-  sortColumnKey: '',
-  sortColumnOrder: 'desc',
-  offset: null,
+  page: page.value,
+  query: filterQuery.value,
+  sortColumnKey: sortColumnKey.value,
+  sortColumnOrder: sortColumnOrder.value,
+  offset: offset.value,
```

Comment:
This object isn't reactive; I'm not sure I understand why we would always return the initial filter query?

Reply:
Yes, it doesn't need to be reactive. We assign ref values to object properties here just so we only need to define values for those parameters once. We use this object to provide fallback values in case any are undefined in `props.initialFetcherParams` object.